zio-http/src/main/scala/zio/http/netty/client/ClientRequestEncoder.scala

```diff
@@ -13,29 +13,31 @@ trait ClientRequestEncoder {
    * Converts client params to JFullHttpRequest
    */
   def encode(req: Request)(implicit trace: Trace): Task[FullHttpRequest] =
-    req.body.asChunk.map { chunk =>
-      val content  = Unpooled.wrappedBuffer(chunk.toArray)
-      val method   = req.method.toJava
-      val jVersion = Versions.convertToZIOToNetty(req.version)
+    ZIO.attempt(Versions.convertToZIOToNetty(req.version)(Unsafe.unsafe)).flatMap { jVersion =>
+      req.body.asChunk.map { chunk =>
```

Comment:
Here we are converting the body to a chunk, and then the chunk to an array, and then the array to a wrapped buffer.

It's better to add `Body#asBuffer` and skip all the intermediates.